### ISSUE TITLE
🤡

### ISSUE THREAD

**Issue by ChrisCates** [rajatonit/emotionToEmoji#1]
Can you determine the emotion of these clowns? 🤡🤡🤡

**Comment by ChrisCates** [comment]
🤡🤡🤡🤡🤡

**Comment by rajatonit** [comment]
Yes! It returned: https://photos.google.com/share/AF1QipOrnE0T4yW1oVps38Y7F7tZTb3-snR-DqQquBOWYP3scH45YcaJCgOArtr9Xj00mw/photo/AF1QipPx7oBDiVSfrkEMKz_Mdh0PUGle_bhCUjX6E2DM?key=Q3AyMnNweEhTSk1jZUEtQ0NxZ0Z1cklSSU9KRkZ3

**Comment by ChrisCates** [comment]
Looks like we got a lot of work to do. 🤣
-- 
*Chris Cates:**"Every decision in your life can be determined in O(log*
*₂(n))"*
*| chriscates.ca <http://chriscates.ca>* | hello@chriscates.ca |
github.com/ChrisCates |
